fizz.go

```diff
@@ -353,6 +353,20 @@ func InputModel(model interface{}) func(*openapi.OperationInfo) {
 	}
 }
 
+// Security adds a security definition to an operation.
+func Security(security *openapi.SecurityRequirement) func(*openapi.OperationInfo) {
```

Comment:
Please add this to the README: https://github.com/wI2L/fizz#operation-informations

Reply:
Also, if we're including security-related helpers functions such as `WithoutSecurity()` and `OptionalSecurity()`, they should also be documented/mentionned.